scripts/generate_test_matrix.py

```diff
@@ -82,14 +81,16 @@
         "import",
         "typed",
     ],
-    "regexp": ["string"],
+    "regexp": ["string", "fieldname"],
```

Comment:
what does 'fieldname' represent?

Reply:
Looks like a regexp in an object key: https://github.com/returntocorp/semgrep/blob/develop/semgrep-core/tests/js/regexp_fieldname.sgrep. It's currently only implemented for JS, but I could see it being used in other languages too (e.g. `dict` key in Python).

Reply:
oh interesting, didn't know that we supported that 😮 , looks good to me!